src/frog-base.tsx

```diff
@@ -168,9 +169,22 @@ export type FrogConstructorParameters<
   verify?: boolean | 'silent' | undefined
 }
 
-export type RouteOptions = Pick<FrogConstructorParameters, 'verify'> & {
-  fonts?: ImageOptions['fonts'] | (() => Promise<ImageOptions['fonts']>)
-}
+export type RouteOptions<M extends string = string> = Pick<
```

Comment:
```suggestion
export type RouteOptions<method extends string = string> = Pick<
```

We can conform to our convention of generics as we aren't working on the Hono type fork (`types/routes.ts`) anymore

Reply:
done!